lochness/redcap/__init__.py

```diff
@@ -320,31 +333,51 @@ class REDCapError(Exception):
 
 
 def redcap_projects(Lochness, phoenix_study, redcap_instance):
-    '''get redcap api_url and api_key for a phoenix study'''
+    '''get redcap api_url and api_key for a phoenix study
+
+    Key Arguments:
+        Lochness: Lochness object.
+        phoenix_study: name of the study, str. eg) STUDY_A
+        redcap_instance: name of the redcap field for the study in the keyring,
+                         str. eg) redcap.STUDY_A
```

Comment:
All right, I accept your change of no `_` requirement in study name. Let's revise the above two `STUDY_A` then.

Reply:
Great - 67c93fe updated the doc string to have `PronetLA` example